src/lib/ext-activitylog.js

```diff
@@ -184,6 +197,10 @@ class View {
   clearTable() {
     this.logView.clearTable();
   }
+
+  renderHeading({ filterTabId }) {
+    this.logHeading.textContent = `Activity Logs Filtered By Tab Id: ${filterTabId}`;
+  }
```

Comment:
This is ok for now, at least it doesn't violate the MVC pattern as we discussed in some of the previous review comments, but after using the panel a big I think that as a user I would expect the "filter by tabId" to be visible along with the other filters (eventually only visible when the activitylog page has detected the filterTabId parameter as part of the url, and maybe without allowing the users to clear the filter as they can on the other filters).

Let's add it as a point in the agenda (so that we can agree if we want to file an issue for this and which priority should it get).